js/modules/views/initialization.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-env browser */
+
+/* global i18n: false */
+
+
+const OPTIMIZATION_MESSAGE_DISPLAY_THRESHOLD = 2000; // milliseconds
+
+// type Canceler = () => Eff Unit
```

Comment:
What is an `Eff`?

Reply:
As briefly discussed `Eff` denotes a native effect, in this case `clearTimeout`. You can learn more about it here: https://github.com/purescript/documentation/blob/5e844f205283c34fc59d9556e2fe1dcb7a738003/guides/Eff.md

I’ll remove this type annotation for now until we had a proper chance to discuss it.